src/storage/accessors/DataAccessor.ts

```diff
@@ -36,6 +36,18 @@ export interface DataAccessor {
    */
   getMetadata: (identifier: ResourceIdentifier) => Promise<RepresentationMetadata>;
 
+  /**
+   * Returns metadata for all resources in the requested container.
+   * These `RepresentationMetadata` objects should only contain metadata
+   * that is relevant for the container representation!
```

Comment:
The way this is phrased, creates some coupling to a component down the line.
Can we explain this more explicitly, maybe with an example?